site/docs/v1/tech/identity-providers/samlv2-idp-initiated/index.adoc

```diff
@@ -0,0 +1,118 @@
+
+[field]#Use NameId for email# [optional]#Optional#::
+If this is enabled, FusionAuth will assume that the `NameID` in the SAML response contains the email address of the user.
+
+[field]#Email claim# [optional]#required# Required::
+The name of the email claim returned in the SAML response.
++
+When [field]#Use NameId for email# is enabled this field will not be displayed and will not be required.
+
+[field]#Verification key# [required]#Required#::
+The public key or certificate that you must import into FusionAuth's KeyMaster. This is the public key provided to you by the identity provider.
+
+=== Options
+
+image::identity-providers/identity-provider-samlv2-idp-initiated-add-options.png[Add SAML v2 Options section,width=1200,role=top-cropped]
+
+==== Form Fields
+
+[api]
+[field]#Reconcile lambda# [optional]#Optional#::
+A lambda maps custom claims returned from the SAML response into the FusionAuth `User` and `Registration`. To learn more about creating a lambda, view the link:/docs/v1/tech/lambdas/samlv2-response-reconcile/[SAML v2 Reconcile lambda documentation].
++
+To configure a lambda, navigate to [breadcrumb]#Settings -> Lambdas#.
+
+[field]#Debug# [optional]#Optional#::
+Some identity providers are not compliant with the SAML and XML signing specifications. This makes it challenging to get them working with FusionAuth. 
++
+If you are running into integration issues, toggle this setting on and FusionAuth will output debugging information into the Event Log during a SAML login. You can find the event log in [breadcrumb]#System -> Event Log#.
+
+== Integration Details
+
+After configuring the Identity Provider, you can view the details to see values that will likely be required by your SAML v2 Identity Provider in order to trust FusionAuth as a relying party. Do so by navigating to [breadcrumb]#Settings -> Identity Providers# and click the green magnifying glass on your SAML provider. 
+
+image::identity-providers/identity-provider-samlv2-idp-initiated-view-list.png[View the identity provider list,width=1200,role=bottom-cropped]
+
+When viewing the details, scroll to the [breadcrumb]#SAML v2 Integration details# section. There you will find the needed values to configure an integration with a SAMLv2 IdP.
+
+=== SAML v2 Integration Details
+
+image::identity-providers/identity-provider-samlv2-idp-initiated-view-details.png[View the SAMLv2 identity provider details,width=1200]
+
+==== Fields 
+
+include::docs/v1/tech/identity-providers/_samlv2-integration-details-fields.adoc[]
+
+== CORS Configuration
+
+In order to complete the login request, the SAML v2 identity provider will make an HTTP `POST` request to the callback URL in FusionAuth. In order for this request to be allowed through the CORS filter you will need to navigate to [breadcrumb]#Settings -> System -> CORS# and add the SAML IdP origin as an [field]#Allowed Origin# the CORS configuration.
```

Comment:
consider
~In order~ To complete the login request, the SAML v2 identity provider will make an HTTP `POST` request to the callback URL in FusionAuth. ~In order for this request to be allowed~ To allow this request through the CORS filter you will need to navigate to [breadcrumb]#Settings -> System -> CORS# and add the SAML IdP origin as an [field]#Allowed Origin# the CORS configuration.